src/AIS_Decoder.cpp

```diff
@@ -2626,7 +2627,11 @@ void AIS_Decoder::OnTimerAIS( wxTimerEvent& event )
     //    Honor the global flag
     if( !g_bAIS_CPA_Alert_Audio )
         m_bAIS_Audio_Alert_On = false;
-
+#ifdef HAVE_PORTAUDIO
```

Comment:
Why only for portaudio?

Reply:
"g_SoundPlayTime" is only set in my Portaudio code, the other Sounds do not use it.